ui/job-view/pushes/PushActionMenu.jsx

```diff
@@ -273,6 +273,20 @@ class PushActionMenu extends React.PureComponent {
               Push Health
             </a>
           </li>
+          <li>
+            <a
+              className="dropdown-item"
+              href={getCompareChooserUrl({
+                newProject: currentRepo.name,
+                newRevision: revision,
+              })}
+              target="_blank"
+              rel="noopener noreferrer"
+              title="Compare Performance"
```

Comment:
Looking ok.

Just please provide a more detailed title, for when the user hovers over it. I think something like "Compare performance against another revision" would be more precise. It's similar to that used in `PerformanceTab` component.